src/SSLLocks.cpp

```diff
@@ -0,0 +1,66 @@
+// Copyright 2005-2017 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#include "murmur_pch.h"
+
+#include "SSLLocks.h"
+
+static QMutex **locks = NULL;
```

Comment:
I'd like to avoid static globals for now. We can't use Q_STATIC_GLOBAL either in 1.3.x, since we need to support Qt 4.
So I've added a destroy() function for now as a compromise. We can revisit and clean stuff up once we can throw away a lot of the legacy baggage.